xtask/src/main.rs

```diff
@@ -39,7 +39,6 @@ fn execute_generate_proto() -> anyhow::Result<()> {
     let proto_files = vec![
         vortex_proto.join("proto").join("dtype.proto"),
         vortex_proto.join("proto").join("scalar.proto"),
-        vortex_proto.join("proto").join("expr.proto"),
```

Comment:
i guess this file doesn't exist anymore

Reply:
Yeah, we need to fix proto serialisation. IIRC there should be exprs but they might have been accidentally deleted